cosmos/config.py

```diff
@@ -207,7 +207,10 @@ class ExecutionConfig:
     :param execution_mode: The execution mode for dbt. Defaults to local
     :param dbt_executable_path: The path to the dbt executable. Defaults to dbt if
     available on the path.
+    :param virtualenv_dir: Directory path to locate the (cached) virtual env that
+    should be used for execution when execution mode is set to `ExecutionMode.VIRTUALENV`
     """
 
     execution_mode: ExecutionMode = ExecutionMode.LOCAL
     dbt_executable_path: str | Path = get_system_dbt()
+    virtualenv_dir: str | Path | None = None
```

Comment:
For sure, I am not too opinionated here, was mostly following the example from `dbt_executable_path` being either a `str` or a `Path` :)

Reply:
I wish we hadn't given users this flexibility since the start 🙈 Now we can't change this interface without breaking users - so I think we can only review the previously existing parameters in Cosmos 2.0